### PR TITLE
Add item user owner data check in Model\Item::isValid

### LINKED ISSUE #9662: Deleted users/accounts are still able to post new items
- [X] I have searched open and closed issues for duplicates

### Bug Description
Two days ago I deleted a spam account from my node. The user account is waiting for permanent deletion. This will happen in 5 days. Today I discovered that there are still new posts coming from that **deleted** account.
It looks like the user has configured the Twitter connector on another Friendica node and is mirroring the spam from there to other Friendica accounts. I other words the deleted account is still automatically producing new posts via mirroring. That's bad. A deleted account shouldn't be able to produce new items.

### Steps to Reproduce

1.  create an account
2.  set up a contact and configure it to mirror posts as your own
3.  delete the account
4.  realize that new mirror posts are still created and published

Actual Result:
The deleted account is still sending new posts during the period of 'waiting for permanent deletion'.

Expected Result:
A deleted account shouldn't be able to post new items. 

### Platform Info

Friendica Version:
2020.12-rc

Friendica Source: 
git

PHP version:
7.4

SQL version:
MariaDB

### PR DESCRIPTION
**Pull request by MrPetovan**
Fixes #9662 

I do not have the necessary setup to test this, but we have a centralized method to post items, right? Right?

### CODE REVIEW

File: src/Model/Item.php [hunk 1]
```diff
@@ -1385,6 +1385,19 @@ public static function isValid(array $item)
...
 		}
 
+		if (!empty($item['uid'])) {
+			$owner = User::getOwnerDataById($item['uid'], false);
+			if (!$owner) {
+				Logger::notice('Missing item user owner data', ['uid' => $item['uid']]);
+				return false;
+			}
+
+			if ($owner['deleted'] || $owner['account_expired'] || $owner['account_removed']) {
```
Comment (annando) [inline src/Model/Item.php]: `deleted` is a field in the `contact` table. The other fields look okay.

Comment (MrPetovan) [reply src/Model/Item.php]: `$owner` comes from the owner view so it would have this field, right? But maybe it isn't relevant?

Comment (annando) [reply src/Model/Item.php]: Yes, it is in the view. But it isn't used in the process of deleting users.

Comment (MrPetovan) [reply src/Model/Item.php]: I'll remove it then.

Comment (MrPetovan) [reply src/Model/Item.php]:
```suggestion
			if ($owner['account_expired'] || $owner['account_removed']) {
```

**Pull request merged**